undebt/cmd/main.py

```diff
@@ -157,57 +121,18 @@ def main():
     args = _handle_arguments()
     logger.setup(args.verbose)  # Reset logging level
 
-    if args.jobs <= 0:
-        log.error('number of processes must be > 0')
-        sys.exit(1)
-
     processor = _file_processor(args.pattern, args.dry_run)
-    files = list(_find_files(args.paths, _fix_exts(args.extension)))
+    files = list(_find_files(args.files))
```

Comment:
We _could_, but the one thing it does is it [prunes non-files](https://github.com/Yelp/undebt/pull/37/files#diff-128f178250d35e92ec0297ddac72bbd4R83). Of course, this could also be achieved with `find`.

/me shrugs